### PR TITLE
Improve call_group documentation

### PR DESCRIPTION
**Pull request by KoBeWi**
- Mentions that `call_group` is deferred.
- Adds code example to `call_group_flags`.

### CODE REVIEW

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
@KoBeWi This needs a rebase, otherwise looks good.

**Comment by KoBeWi** [comment]
Turns out that the conflict was with another PR that did almost the same #48305 heh

**Comment by KoBeWi** [comment]
CC @Calinou

**Comment by akien-mga** [comment]
I thought we had discussed recently to actually make `call_groups` real-time by default, but I can't find any pending, merged or closed PR about it. Maybe it was only on chat.

**Comment by Calinou** [comment]
> I thought we had discussed recently to actually make `call_groups` real-time by default, but I can't find any pending, merged or closed PR about it. Maybe it was only on chat.

See https://github.com/godotengine/godot-proposals/issues/2671.

**Review by akien-mga — APPROVED** [review]

File: doc/classes/SceneTree.xml [hunk 1]
```diff
@@ -29,7 +29,9 @@
 			<argument index="2" name="method" type="StringName" />
 			<description>
 				Calls [code]method[/code] on each member of the given group, respecting the given [enum GroupCallFlags]. You can pass arguments to [code]method[/code] by specifying them at the end of the method call.
-				[b]Note:[/b] Group call flags are used to control the method calling behavior. If the [constant GROUP_CALL_REALTIME] flag is present in the [code]flags[/code] argument, methods will be called immediately. If this flag isn't present in [code]flags[/code], methods will be called with a one-frame delay in a way similar to [method call_group].
```
Comment (akien-mga) [inline doc/classes/SceneTree.xml:32]: Was this removal intentional BTW?

Comment (KoBeWi) [reply doc/classes/SceneTree.xml:32]: Yes, the code example is enough.

**Comment by akien-mga** [comment]
Cherry-picked for 3.3.4.
